app/controllers/reviews_controller.rb

```diff
@@ -1,2 +1,42 @@
 class ReviewsController < ApplicationController
+  before_action :set_book, only: %i[edit update]
```

Comment:
Not sure why these (review controller, review model, show book view, review edit view) change are shown here.
I guess you need to merge or rebase with `feature/show-book` branch again